libs/packages/components/src/lib/video-player/video-player.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit, Input, AfterViewInit, ViewEncapsulation } from '@angular/core';
+import { InitPxVideo } from '../video-player/js/px-video';
+
+@Component({
+  selector: 'sds-video-player',
+  templateUrl: './video-player.component.html',
+  styleUrls: ['./css/px-video.css'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class SdsVideoPlayerComponent implements OnInit, AfterViewInit {
+  VvtFileURL: string;
+  videoHeightEl: string;
+  videoWidthEl: string;
+  videoPosterEl: string;
+  videoPreloadEl: string;
+  videoSourceMp4El: string;
+  videoSourceWebmEl: string;
+  videoCaptionEl: string;
+  imageSrcEl: string;
+  videoIdEl: string;
+  GLOBAL_STRINGS: any;
+
+  @Input() videoSourceWebm: string;
+  @Input() videoSourceMp4: string;
+  @Input() videoHeight: string;
+  @Input() videoWidth: string;
+  @Input() videoPlayerId: string;
+  @Input() videoCaption: string;
+  @Input() videoPoster: string;
+  @Input() videoSeekInterval: number;
+  @Input() videoDebug: boolean;
+  @Input() videoPreload: string;
+  @Input() imageSrc: string;
+  @Input() videoTitle: string;
+  @Input() captionOnDefault: boolean;
```

Comment:
Consider making this a configuration object with a strict typing of the values you expect.

Reply:
Created object and it check values